barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/mock_circuits.hpp

```diff
@@ -60,19 +60,5 @@ class MockCircuits {
             builder.create_big_add_gate({ a_idx, b_idx, c_idx, d_idx, FF(1), FF(1), FF(1), FF(-1), FF(0) });
         }
     }
-
-    /**
-     * @brief Populate a builder with some arbitrary goblinized ECC ops
-     *
-     * @param builder
-     */
-    static void construct_goblin_ecc_op_circuit(GoblinUltraCircuitBuilder& builder)
```

Comment:
Maybe we should keep this function but expand it to include at least one of each type of op (mul, add, eq) and use this in the Goblin-specific places I mentioned to continue to ensure we're getting robust testing that's not implicitly tied to the _add_gates_to_ensure functionality.